tests/test_main.py

```diff
@@ -637,6 +637,7 @@ def test_complicated(tmpdir):
     )
     stdout = stdout.replace('Hello, "world"\n\n\n\n', 'Hello, "world"\n\n\n')
     stdout = stdout.replace('</html>\n\n\n', '</html>\n\n')
+    stdout = re.sub(r'Content-Id:.*', '', stdout)
```

Comment:
Is this because `Content-Id` is automatically generated?

Reply:
Yes that sounds right. I think it's okay here since the `Content-ID` header is tested in `test_template_message.py`.